app/jobs/delete_user_job.rb

```diff
@@ -0,0 +1,17 @@
+class DeleteUserJob < ApplicationJob
+  queue_as :default
+
+  def perform(user)
```

Comment:
I got rid of ActiveRecord transactions here because destroys are always put within a transaction - from the docs.
And we only ever destroy here.

So far, regular users don't have anything besides access to family accounts so other cleanup might be a future thing.

Admin deletion is currently in two
- the last admin and we just get rid of the family which deletes the admin user
- not the last admin and we just delete the user only - and related data.